docs.json

```diff
@@ -100,6 +102,7 @@
         "chrome": "https://chrome.google.com/webstore/detail/pieces-save-code-snippets/igbgibhbfonhmjlechmeefimncpekepm",
         "jetbrains": "https://plugins.jetbrains.com/plugin/17328-pieces--save-search-share--reuse-code-snippets",
         "vscode": "https://marketplace.visualstudio.com/items?itemName=MeshIntelligentTechnologiesInc.pieces-vscode",
+        "obsidian": "https://obsidian.md/plugins?id=obsidian-pieces",
```

Comment:
is this just a search query for the plugins page? clever solution but im curious if there is going to be a true page link later? @randy-at-pieces

Reply:
alright ill keep an eye out - when clicking this it takes you to a search field on obsidian thats all lol